client/client.go

```diff
@@ -127,3 +127,42 @@ func (client *Client) SysInfo() (*SysInfo, error) {
 
 	return &sysInfo, nil
 }
+
+// Capability holds information about a capability that a snap may request
+// from a snappy system to do its job while running on it.
+type Capability struct {
+	// Name is a key that identifies the capability. It must be unique within
```

Comment:
I've documented them in rest.md and I've kept the copy descriptions in the client package. Feel free to object and I'll have those removed though we'll still need _something_ since it's a part of the public API.